src/frontend/assets/zoom-logo.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg fill="#5e57e9" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 29.31 25.71">
```

Comment:
I think the `fill` attribute should be removed to be consistent with the other logos.

Reply:
I can do that, but this is the exact svg [you provided for Lutris](github.com/lutris/lutris-runtime/pull/8). Feel free to provide a suggestion for this file and I will accept it.